algebraic-types/rs2bril/Makefile

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
I know Makefiles are cool, but these are kinda the standard commands one would run for any Cargo-based Rust project. So maybe not necessary?